packages/CodingStandard/tests/Issues/Issue855Test.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Tests\Sniffs\Issues;
+
+use Symplify\EasyCodingStandardTester\Testing\AbstractCheckerTestCase;
+
+final class Issue855Test extends AbstractCheckerTestCase
+{
+    public function test(): void
+    {
+        $this->doTestWrongToFixedFile(__DIR__ . '/wrong/wrong855.php.inc', __DIR__ . '/fixed/fixed855.php.inc');
```

Comment:
If they're identical, you can use `$this->doTestCorrectFile()`

Reply:
Ok, updated